ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -99,6 +105,8 @@ defaultChainDbView chainDB = ChainDbView {
     , getHeaderStateHistory = ChainDB.getHeaderStateHistory chainDB
     , getPastLedger         = ChainDB.getPastLedger         chainDB
     , getIsInvalidBlock     = ChainDB.getIsInvalidBlock     chainDB
+    , getIsFetched          = ChainDB.getIsFetched          chainDB
```

Comment:
Need to think more carefully: what happens if the block of interest is outside of the `k from tip` limit mentioned on the `ChainDB.getIsFetched` comment?